packages/math/typesetter.lua

```diff
@@ -149,6 +151,18 @@ local function handleMath (_, mbox, mode)
     end)
     SILE.typesetter:endline()
     SILE.typesetter:pushExplicitVglue(SILE.settings:get("math.displayskip"))
+  elseif mode == "numbered" then
+    SILE.typesetter:endline()
+    SILE.typesetter:pushExplicitVglue(SILE.settings:get("math.displayskip"))
+    SILE.typesetter:pushGlue(SILE.nodefactory.hfillglue())
+    SILE.typesetter:pushHorizontal(mbox)
+    SILE.typesetter:pushGlue(SILE.nodefactory.hfillglue())
+    SILE.call("increment-counter", { id="equation" })
+    SILE.typesetter:typeset("(")
+    SILE.call("show-counter", { id="equation" })
+    SILE.typesetter:typeset(")")
```

Comment:
maybe we should move the formatting in a hook command, so that users or 3rd-party package could override it?
(If going with options as suggested in my other command, those could also be passed through to the hook)
This could offer a customization point for styling differently, rather than hard-code the parentheses in that low level function. (eg. add color or font style; subscribe to label refs, etc.)

Reply:
I agree with that. My initial intention was just to provide a quick mean to get what I almost always need, and not dealing with the plethora of variations in equations numbering. Letting anyone be free to achieve what they want (a numbering like "[chapter.equation]' comes to my mind) would be best. I'll see what I can do, but I'll probably need some help ;)